src/plugins/injection/README.md

```diff
@@ -323,3 +324,23 @@ These errors are set with `inject/limit/min` and `inject/limit/max` and one of b
         ```
         contrast = 500
         ```
+        
+## Logging of changes
+
+To see which changes were applied to the whole KeySet, additional metadata will be written to the rootkey.
```

Comment:
Description is very short, it is not yet clear how this logging should work. In METADATA.ini logging is already proposed, maybe you can extend this work so that logging is not specific to the inject plugin?

Reply:
Maybe `log` is the wrong wording as I just want to backtrace injections which were done (i.e. typo error on setting xy, structure error on key zx, etc.)

Reply:
> I do not see anything concerning logging in the metadata.ini

Sorry, it was removed in 06e10ca90ea129cf0712d8fca40f8f1159616174 and did not really contain useful information.

> Maybe log is the wrong wording as I just want to backtrace injections which were done

Yes, better to change the wording then.

Reply:
And why not add the changes directly to the keys where the changes occurred?

Reply:
> And why not add the changes directly to the keys where the changes occurred?

Because some injections simply remove a whole configuration/ section. It is also much simpler to use when all changes can be found on one key's metadata than searching for all keys if they have an injection on them imo